src/main/java/org/candlepin/resource/CloudRegistrationResource.java

```diff
@@ -288,4 +298,23 @@ private void validateCloudAuthenticationResult(CloudAuthenticationResult result)
             throw new NotAuthorizedException(errmsg);
         }
     }
+
+
+    @Override
+    @Transactional
+    public void deleteAnonymousConsumersByAccountId(String cloudAccountId) {
```

Comment:
Can we add a javadoc here explaining this endpoint is only for testing purposes?